packages/corvid-cli/test/clone.e2e.js

```diff
@@ -401,4 +402,34 @@ describe("clone", () => {
       );
     });
   });
+  describe("Failed clone", () => {
+    it("should clean files", async () => {
+      const paths = ["src", ".corvid"];
+      editorServer = await localFakeEditorServer.start();
+      const { tempDir, siteUrl } = await setupSuccessfullClone();
```

Comment:
it's confusing, because you are trying to set up a failing clong

Reply:
done